packages/server/src/seed/seed.controller.ts

```diff
@@ -32,6 +33,10 @@ export class SeedController {
     await this.seedService.deleteAll(OfficeHourModel);
     await this.seedService.deleteAll(QuestionModel);
     await this.seedService.deleteAll(QueueModel);
+    await this.seedService.deleteAll(UserCourseModel);
+    await this.seedService.deleteAll(UserModel);
+    await this.seedService.deleteAll(CourseModel);
+    await this.seedService.deleteAll(EventModel);
```

Comment:
I think we need to place `EventModel` above `UserModel` because of foreign key contraints